ui/app/services/store.js

```diff
@@ -186,20 +177,18 @@ export default Store.extend({
       dataset: array,
     };
     this.setLazyCacheForModel(modelName, query, dataSet);
-  },
+  }
 
   clearDataset(modelName) {
-    const cacheList = this.lazyCaches;
-    if (!cacheList.size) return;
-    if (modelName && cacheList.has(modelName)) {
-      cacheList.delete(modelName);
+    if (!this.lazyCaches.size) return;
+    if (modelName && this.lazyCaches.has(modelName)) {
+      this.lazyCaches.delete(modelName);
       return;
     }
-    cacheList.clear();
-    this.set('lazyCaches', cacheList);
-  },
+    this.lazyCaches.clear();
```

Comment:
Do we still need to set `lazyCaches`to `cacheList` here?

Reply:
Nope, since it's now a [TrackedMap](https://github.com/tracked-tools/tracked-built-ins) we don't need to set it to itself anymore to trigger changes :)

Reply:
ohh awesome!!! :D